tornettools/parse_oniontrace.py

```diff
@@ -62,9 +62,6 @@ def __extract_relay_tput(args, data, startts, stopts):
 def __get_perfclient_cbt(data, circuittype, startts, stopts):
     perf_cbt = []
 
-    # cbts can differ by microseconds
-    resolution = 1.0 / 1000000.0
```

Comment:
I don't think we want to remove resolution (here are anywhere in this PR), it should instead be correctly incorporated into the calculation of the error (ideally resolution error would be incorporated into the calculation of the shaded regions on the CDFs for every metric).

This seems like a separate issue to fix, so perhaps here we just comment the resolution and add a TODO, and then we come back to fix it in a separate PR.

Reply:
Done